test/testdata/schema/ghc92/vscode-extension-schema.golden.json

```diff
@@ -249,6 +249,622 @@
         "scope": "resource",
         "type": "boolean"
     },
+    "haskell.plugin.semanticTokens.config.class": {
```

Comment:
I don't know whether we might want more configuration options later, but perhaps it would be wise to nest all these in `config.tokenMapping` or something?

Reply:
I guess this is the main remaining question. Not sure if it's going to make that much of a difference or not :thinking: 

I guess the alternative would be to suffix the option name with `Token`? So we could have `config.classToken` which is relatively clear what it's doing...

Reply:
> I don't know whether we might want more configuration options later, but perhaps it would be wise to nest all these in config.tokenMapping or something?

I think this one would be better. I am going to extend the properties api to better support nested configuration.